xbmc/pvr/channels/PVRChannelGroupInternal.cpp

```diff
@@ -111,7 +112,7 @@ std::vector<std::shared_ptr<CPVRChannelGroupMember>> CPVRChannelGroupInternal::
     RemoveDeletedGroupMembers(
         const std::vector<std::shared_ptr<CPVRChannelGroupMember>>& groupMembers)
 {
-  const std::vector<std::shared_ptr<CPVRChannelGroupMember>> removedMembers =
+  std::vector<std::shared_ptr<CPVRChannelGroupMember>> removedMembers =
```

Comment:
Not saying this is wrong, but could you explain why this change is needed?

Reply:
My take on it is removeMembers is returned to the caller, and thats non const. So either the return type goes const, or why bother keeping this one const if it has to be cast away by the compiler anyway

Reply:
Thanks, that's what I thought.

Reply:
If this variable is const, it cannot be moved at the return statement.

see https://clang.llvm.org/extra/clang-tidy/checks/performance-no-automatic-move.html